go/pkg/command/command.go

```diff
@@ -45,6 +45,33 @@ func (s InputType) String() string {
 	return fmt.Sprintf("InvalidInputType(%d)", s)
 }
 
+// SymlinkBehavior represents how symlinks are handled.
+type SymlinkBehaviorType int
+
+const (
+	// UnspecifiedSymlinkBehavior means following clients.TreeSymlinkOpts
+	// or DefaultTreeSymlinkOpts if clients.TreeSymlinkOpts is null.
+	UnspecifiedSymlinkBehavior SymlinkBehaviorType = iota
```

Comment:
no needs to be public?

Reply:
I am not so comfortable to reveal a value nobody outside of the package can compare.  Let me defer to the repository owner for the decision.

Reply:
I think this being public is fine and consistent with the rest of the file.